packages/react-native-sdk/docusaurus/docs/reactnative/02-tutorials/02-audio-room.mdx

```diff
@@ -1,9 +1,1111 @@
 ---
 title: Audio Room Tutorial
-description: How to build an audio room using Stream's video SDKs
+description: How to build an audio room using Stream's React Native Video SDK
 ---
 
-:::info
-This guide is still in progress and will be updated soon.
-If you have any questions, please reach out to us on our [website](https://getstream.io/video/#contact).
+import { TokenSnippet } from '../../../shared/_tokenSnippet.jsx';
+
+This tutorial will teach you how to build an audio room experience like Twitter Spaces or Clubhouse.
+The end result will look like the image below and will support the following features:
+
+- Backstage mode. You can start the call with your co-hosts and chat a bit before going live
+- Calls run on Stream's global edge network for optimal latency and scalability
+- There is no cap to how many listeners you can have in a room
+- Listeners can raise their hand, and be invited to speak by the host
+- Audio tracks are sent multiple times for optimal reliability
+
+![Preview of the final result](../assets/02-tutorials/02-audio-room/audio-room-target.png)
+
+Time to get started building an audio room for your app.
+
+### Step 1 - Setup a new React Native app
+
+Create a new React Native app using the official template,
+
+```bash title=Terminal
+npx react-native@latest init StreamAudioRoomExample
+cd StreamAudioRoomExample
+```
+
+Please follow through the __[SDK installation](../../setup/installation)__ steps to properly setup the SDK and its prerequisites. 
+### Step 2 - Add the Audio room screen component
+
+Let us first add our constants that will be reused in multiple files later. Create a new file `src/constants.ts` and add the following code:
+
+```tsx title="src/constants.ts"
+export const apiKey = 'REPLACE_WITH_API_KEY'; // the API key can be found in the "Credentials" section
+export const token = 'REPLACE_WITH_TOKEN'; // the token can be found in the "Credentials" section
+export const userId = 'REPLACE_WITH_USER_ID'; // the user_id can be found in the "Credentials" section
+export const callId = 'REPLACE_WITH_CALL_ID'; // the call_id can be found in the "Credentials" section
```

Comment:
I don't see we are using this exported `callId` anywhere on our tutorial. So we can probably remove it.

Reply:
nice catch